docs/specs/aerogear-sync-use-cases.asciidoc

```diff
@@ -1,5 +1,6 @@
 ---
-layout: basic
+layout: post
+section: docs
 title: AeroGear Data Sync
```

Comment:
Preliminary check `ack aerogear-sync-use-cases` found that this is not really linked.

Remove it OR  link it from Docs index?